.eslintrc.json

```diff
@@ -17,10 +23,14 @@
     }
   },
   "rules": {
-    "max-len": [
+    "@typescript-eslint/interface-name-prefix": [
       "error",
-      { "code": 80, "ignoreTemplateLiterals": true, "ignoreUrls": true }
+      { "prefixWithI": "always" }
     ],
-    "no-tabs": "error"
+    "no-tabs": "error",
+    "react/prop-types": "off", // Props should be described as TS types
+    "@typescript-eslint/explicit-function-return-type": "off",
+    // TODO: remove after rewriting code to TS
```

Comment:
is this PR still WIP? we don't merge TODOs usually

Reply:
Yes, it's still WIP and will be few more days at least. As comment says this TODO will be removed after migrating all components to TS